src/project/listeners/key_listener.cpp

```diff
@@ -0,0 +1,24 @@
+#include "listeners/key_listener.hpp"
+
+void KeyListener::OnKeyPressed(const std::shared_ptr<KeyEvent>& event, const std::map<Event, std::vector<ActionType>>& actionTypeKeyBinds) {
+    auto it = actionTypeKeyBinds.find(*event);
+
+    if (it != actionTypeKeyBinds.end()) {
+        for (const ActionType& actionType : it->second) {
+            const ActionType enumOfType = ActionType::convertStringToEnum(actionType.getName());
+
+            if (enumOfType == ActionType::JUMP) {
+            } else if (enumOfType == ActionType::GODOWN) {
+            } else if (enumOfType == ActionType::GOUP) {
+            } else if (enumOfType == ActionType::GOLEFT) {
+            } else if (enumOfType == ActionType::GORIGHT) {
+            } else if (enumOfType == ActionType::PAUSE) {
+            } else {
+            }
```

Comment:
Maak hier aub een switch van:

```suggestion
            switch (enumOfType) {
                case ActionType::JUMP:
                    break;
                case ActionType::GODOWN:
                    break;
                case ActionType::GOUP:
                    break;
                case ActionType::GOLEFT:
                    break;
                case ActionType::GORIGHT:
                    break;
                case ActionType::PAUSE:
                    break;
                default:
                    break;
            }
```